packages/eslint-config/src/configs/typescript.ts

```
@@ -0,0 +1,63 @@
+      parser: typescriptParser,
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true
+        },
+        project: options?.project,
+        tsconfigRootDir: options?.tsconfigRootDir,
+        sourceType: 'module'
+      }
+    },
+    rules: {
+      ...typescriptPlugin.configs['recommended-type-checked']!.rules,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)